definitions/ext-rf_scanner/dashboard.json

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
The PR is failing the dashboard validation, but it'll get fixed if you sanitize it using:

```
docker-compose run sanitize-dashboards
```